include/deal.II/base/thread_local_storage.h

```diff
@@ -175,6 +176,23 @@ namespace Threads
     T &
     get(bool &exists);
 
+    /**
+     * If the thread with given `id` has an object currently stored, then return
+     * it by value via the `std::optional` object. If the indicated thread does
+     * not have an object stored, return an empty `std::optional`.
+     *
+     * Note that in the successful case, this function returns a *copy* of
+     * the object, unlike get() which returns a reference to it. This is
+     * because when you call get(), you are calling it from the current
+     * thread (i.e., the thread that "owns" the object), and so all accesses
+     * are by definition not concurrent. On the other hand, if you are asking
+     * about the object owned by a different thread, that other thread my
```

Comment:
```suggestion
     * about the object owned by a different thread, that other thread might
```

Reply:
I meant to say "may", "might" might even be better. So fixed.